src/main/java/io/fabric8/maven/docker/util/EnvUtil.java

```diff
@@ -78,34 +81,54 @@ public static boolean greaterOrEqualsVersion(String versionA, String versionB) {
         return largerVersion != null && largerVersion.equals(versionA);
     }
 
+    private static final Function<String, String[]> SPLIT_ON_LAST_COLON = new Function<String, String[]>() {
+        @Override
+        public String[] apply(String element) {
+          int colon = element.lastIndexOf(':');
+          if (colon < 0) {
+              return new String[] {element, element};
+          } else {
+              return new String[] {element.substring(0, colon), element.substring(colon + 1)};
```

Comment:
what if colon is the last character in `element` ? wouldn't then `element.substring(colon+1)` throw an error here ?